src/main/kotlin/io/zerobase/smarttracing/Main.kt

```diff
@@ -18,9 +19,10 @@ import javax.servlet.DispatcherType
 import javax.servlet.FilterRegistration
 
 typealias MultiMap<K,V> = Map<K, List<V>>
+typealias EmailConfig = Map<EmailType, EmailConfigEntry>
 
-data class AmazonConfig(val from: String)
-data class Config(val database: Neo4jConfig, val amazonaws: AmazonConfig, val siteTypeCategories: MultiMap<String, String>, val scannableTypes: List<String>): Configuration()
+data class AWS(val fromEmailSES: String)
```

Comment:
i think you misunderstood a little bit. the email we send from isn't SES or AWS config. it's just app config. SES config would be things like which region to use or an alternate endpoint instead of AWS to use (useful for testing). i would structure it something like this:

```kotlin
data class SimpleEmailService(val region: String = 'us-east-1', val endpoint: URI? = null)
data class AwsConfig(val ses: SimpleEmailService = SimpleEmailService())
```

that way, we can add other service configs into the aws config when we need to. the yaml would look like this:

```yaml
aws:
  ses:
    region: us-east-1
```